prow/config/config.go

```diff
@@ -362,6 +374,30 @@ func loadConfig(prowConfig, jobConfig string) (*Config, error) {
 	return nc, nil
 }
 
+// Loads multiple paths of secrets and add them in a map.
+func LoadSecrets(paths []string) (map[string][]byte, error) {
+	secretsMap := make(map[string][]byte)
```

Comment:
If you're going to use `make` here you can size it as well to the length of the paths

Reply:
How much length should be ok? The hook is using 3 paths. Is there any other binary that is using more?